src/transport.js

```diff
@@ -53,11 +54,18 @@ module.exports = function (swarm) {
       // specific for the transport we are using
       const proxyConn = new Connection()
 
-      next(multiaddrs.shift())
+      each(multiaddrs, next)
```

Comment:
Oh and i don't want you to slow down on the relay as it will be game changing for my project!!